manifests/helm-profiles/compatibility-version-1.20.yaml

```diff
@@ -1,6 +1,7 @@
 pilot:
   env:
     # 1.21 behavioral changes
+    ENABLE_ENHANCED_RESOURCE_SCOPING: "false"
```

Comment:
nit: this is a1.22 change, so the comment on L3 doesn't apply

Reply:
Right, changed.